jupiterone/internal/client/question.go

```diff
@@ -43,6 +43,7 @@ func (c *JupiterOneClient) GetQuestion(id string) (*Question, error) {
 				title
 				description
 				queries {
+					name
```

Comment:
The client was missing the `name` property. I've added it so that the client receives it  and can pass it back to the tf state.